src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx

```diff
@@ -25,7 +26,7 @@ describe("SendFeedback", () => {
     ).toBeInTheDocument()
   })
 
-  it("submits the form", async () => {
+  it("submits the form (not logged-id user)", async () => {
```

Comment:
```suggestion
  it("submits the form (not logged-in user)", async () => {
```

Reply:
oh, I made a typo again 😩. thank you for noticing!

Reply:
corrected